go/chat/utils/utils.go

```diff
@@ -1235,7 +1235,7 @@ func formatVideoDuration(ms int) string {
 	return fmt.Sprintf("%d:%02d", minutes, seconds)
 }
 
-func formatVideoSize(bytes int64) string {
+func PresentBytes(bytes int64) string {
```

Comment:
could just use `humanize.Bytes` throughout

Reply:
I already have this.